vm.go

```diff
@@ -225,6 +263,31 @@ func (vm *VM) EvaluateSnippetMulti(filename string, snippet string) (files map[s
 	return
 }
 
+// FindDependencies finds all transitive dependencies (import and importstr) of a snippet of
+// Jsonnet code and returns a sorted array of unique dependencies.
+//
+// The filename parameter is only used for error messages.
+func (vm *VM) FindDependencies(filename string, snippet string) ([]string, error) {
```

Comment:
Makes sense. I will remove support for `--exec` and `stdin`, and instead pass all the input files to `FindDependencies` where I can loop over them while keeping a common dependencies map.

I have a few questions:

> taking a filePath (or multiple filePaths) and importedFrom here

What would the value of `importedFrom` be for the input files? If it is always some dummy file in the current working directory, can it be dropped from the arguments to `FindDependencies`?

> This would give us the canonical path

The current implementation of `Import` (for FIleImporter) seems to return relative paths when either `importedFrom` or `--jpath` is a relative path. However, looking at `importer.tryPath()` which returns absPath, the expectation is that it should return an absolute path. So, should I fix `importer.tryPath()` to actually return the absolute path?

Reply:
The current `Import` implementation also treats `/a/b/test.libsonnet` different from `/a/b/c/../test.libsonnet`, but they are actually the same file. `filepath.Abs()` handles this case, but doesn't work with symlinks.

Reply:
> What would the value of importedFrom be for the input files? If it is always some dummy file in the current working directory, can it be dropped from the arguments to FindDependencies? 

It should probably be `""` (an empty string). I believe it will be interpreted as CWD. Eventually, we want to generalize "" as "top-level-import", per #329.

I would keep the `importedFrom` in FindDependencies and pass `""` from cmd. This is consistent with other functions and provides some extra flexibility.

> The current implementation of `Import` (for FIleImporter) seems to return relative paths when either `importedFrom` or `--jpath` is a relative path. However, looking at `importer.tryPath()` which returns absPath, the expectation is that it should return an absolute path. So, should I fix `importer.tryPath()` to actually return the absolute path?

You're right and I didn't notice that before. It should always return the absolute path.